src/components/fields/MultiTagArrayField.js

```diff
@@ -0,0 +1,163 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import BaseComponent from "../BaseComponent";
+import {classNames, getUiOptions} from "../../utils";
+import ReactContext from "../../ReactContext";
+import {Affix, Button} from "../components";
+
+
+@BaseComponent
+export default class MultiTagArrayField extends React.Component {
+	static contextType = ReactContext;
+	static propTypes = {
+		uiSchema: PropTypes.shape({
+			"ui:options": PropTypes.shape({
+				buttons: PropTypes.arrayOf(PropTypes.oneOfType(
+					[
+						PropTypes.shape({
+							operation: PropTypes.oneOf(["moveAll"]),
+							fromField: PropTypes.string.isRequired,
+							toField: PropTypes.string.isRequired,
+							label: PropTypes.string,
+							className: PropTypes.string
+						}),
+						PropTypes.shape({
+							operation: PropTypes.oneOf(["move"]),
+							toField: PropTypes.string.isRequired,
+							label: PropTypes.string,
+							className: PropTypes.string
+						}),
+						PropTypes.shape({
+							operation: PropTypes.oneOf(["delete"]),
+							label: PropTypes.string,
+							className: PropTypes.string
+						})
+					]
+				))
+			})
+		}),
+		schema: PropTypes.shape({
+			type: PropTypes.oneOf(["object"])
+		}).isRequired,
+		formData: PropTypes.object
+	}
+
+	constructor(props) {
+		super(props);
+		this.state = {activeButtonIdx: undefined};
+	}
+
+	setContainerRef = (elem) => {
+		this.containerElem = elem;
+	}
+
+	getContainerRef = () => {
+		return this.containerElem;
+	}
+
+	render() {
+		const SchemaField = this.props.registry.fields.SchemaField;
+		const {Row, Col} = this.context.theme;
+		const {schema, uiSchema, idSchema, errorSchema, formData = {}} = this.props;
+		const uiOptions = getUiOptions(uiSchema);
+		const {buttons = []} = uiOptions;
+
+		const {activeButtonIdx} = this.state;
+
+		const propertyKeys = Object.keys(schema.properties)
+
+		return (
+			<Row className={`laji-form-multi-tag-array-field${activeButtonIdx !== undefined ? " laji-form-multi-tag-array-field-active" : ""}`}>
+				<Col xs={3} sm={3} md={2} lg={2} className={"laji-form-multi-tag-array-field-buttons"} ref={this.setContainerRef}>
+					<Affix getContainer={this.getContainerRef}
+						   topOffset={this.props.formContext.topOffset + 15}
+						   bottomOffset={this.props.formContext.bottomOffset}>
```

Comment:
Minor thing, but for aligning JSX props I recommend either having everything simply tabbed (I'll use __ for one tab and . for one space):

```
__<Tag
____prop="1"
____prop="2"
____prop="3"
```

Or having the first prop inline, then aligning the rest of the props by indenting with tabs and aligning with spaces. Personally I prefer this :man_shrugging: 

```
__<Tag prop="1"
__.....prop="2"
__.....prop="3"
```